devchat.go

```diff
@@ -45,16 +46,16 @@ var (
 	rtm       *slack.RTM
 	client    = loadTwitterClient()
 
-	red      = color.New(color.FgHiRed)
-	green    = color.New(color.FgHiGreen)
-	cyan     = color.New(color.FgHiCyan)
-	magenta  = color.New(color.FgHiMagenta)
-	yellow   = color.New(color.FgHiYellow)
-	blue     = color.New(color.FgHiBlue)
-	black    = color.New(color.FgBlack)
-	white    = color.New(color.FgHiWhite)
-	colorArr = []*color.Color{yellow, cyan, magenta, green, white, blue, red}
-
+	red         = color.New(color.FgHiRed)
+	green       = color.New(color.FgHiGreen)
+	cyan        = color.New(color.FgHiCyan)
+	magenta     = color.New(color.FgHiMagenta)
+	yellow      = color.New(color.FgHiYellow)
+	blue        = color.New(color.FgHiBlue)
+	black       = color.New(color.FgBlack)
+	white       = color.New(color.FgHiWhite)
+	colorArr    = []*color.Color{yellow, cyan, magenta, green, white, blue, red}
```

Comment:
okie so we can rename this variable to `colorMap` and make it a `map[string]*color.Color`

Reply:
nope, I'm wrong